source/CommonJobs/CommonJobs.Application.EvalForm/DTOs/EmployeeEvaluationDTO.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm
+{
+    public class EmployeeEvaluationDTO
+    {
+        public bool IsResponsible { get; set; }
+
+        public string FullName { get; set; }
+
+        public string CurrentPosition { get; set; }
+
+        public string Seniority { get; set; }
+
+        public List<string> Evaluators { get; set; }
+
+        public EvaluationState State { get; set; }
+    }
+
+    public enum EvaluationState
+    {
+        InProgress,
+        WaitingAuto,
+        WaitingResponsible,
+        WaitingCompany,
```

Comment:
Once the responsible's done and  the auto-evaluation is done (no need to wait for the evaluators but it will be preferred), the responsible can proceed to generate the company evaluation. This evaluation will have an average of the responsible's and evaluators califications and can be edited to reach a final number so to speak. This one will be the evaluation that the employee will see next to his when in the feedback meeting.